static/js/common.js

```diff
@@ -1063,4 +1063,4 @@ function updateQueryModeUI(queryMode) {
         $('#query-mode-options #mode-option-2').addClass('active');
         $('#query-mode-btn span').html('Code');
     }
-}
+}
```

Comment:
Revert this change

Reply:
Actually, this may be from an automatic formatter that's part of our PR pipeline. If that's the case you can ignore my comment and keep this change. But if this is not from the automatic formatting, you should revert it